saskatoon/fixtures/harvest-property.json

```diff
@@ -51,7 +52,8 @@
     "is_active": true,
     "authorized": true,
     "pending": true,
-    "pending_contact_name": "Nouveau Proprio",
+    "pending_contact_first_name": "Nouveau",
+    "pending_contact_family_name": "Proprio",
```

Comment:
that's good! Have you tried to load the fixture with `python saskatoon/manage.py loaddata saskatoon/fixtures/harvest-property.json` ?

Reply:
please also test `saskatoon.json`

Reply:
@2ynn I haven't tested `python saskatoon/manage.py loaddata saskatoon/fixtures/harvest-property.json` yet. I'm checking and letting you know about this